tests/sles4sap/migrate_sles_to_sles4sap.pm

```diff
@@ -36,6 +36,7 @@ sub run {
     wait_serial "Please enter your activation code", timeout => 5;
     type_string "${regcode}\n";
     assert_script_run "ls /tmp/OK" || die "Migration failed!";
```

Comment:
Not related to this PR: `die` will never be reached here, as `assert_script_run` will fail and stop the test (and the whole test, because of `fatal => 1` in `test_flags`).

(Sorry, I didn't see this last week...)